src/cluster/cluster.cc

```diff
@@ -37,11 +39,11 @@
 #include "time_util.h"
 
 ClusterNode::ClusterNode(std::string id, std::string host, int port, int role, std::string master_id,
-                         std::bitset<kClusterSlots> slots)
+                         std::bitset<kClusterSlots> const &slots)
```

Comment:
I think in this project we usually put the `const` qualifier before the type:
`const std::bitset<kClusterSlots> &slots`